src/core/listener.c

```diff
@@ -334,7 +334,8 @@ MsQuicListenerStart(
     Status =
         QuicLibraryGetBinding(
             &UdpConfig,
-            &Listener->Binding);
+            &Listener->Binding
+        );
```

Comment:
```suggestion
            &Listener->Binding);
```
Please don't change the style.